include/glow/Optimizer/Optimizer.h

```diff
@@ -43,6 +43,12 @@ void optimize(Function *F, CompilationMode mode);
 /// operators.
 void lower(Function *F, const Backend &B);
 
+/// \returns a pair of a bool and a vector of the component NodeInfo of nodes
+/// that would be lowered from NI. If no nodes would be lowered from this node,
+/// then the bool is set to false.
```

Comment:
Why is the bool needed? Why not just return the empty vector?

Reply:
Some nodes are lowered by replacing them with their input, e.g. the `RegressionNode` and `AddGradNode`. So we want to say that the Node can be lowered and that it is lowered into nothing.